src/components/OutdoorWeatherWidget.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react";
+import { ReactComponent as Sun } from "../icons/sun.svg";
+import { ReactComponent as Cloudy } from "../icons/sun.svg";
+import { ReactComponent as PartlyCloudy } from "../icons/cloud-sun.svg";
+import { ReactComponent as Rain } from "../icons/cloud-rain.svg";
+import { ReactComponent as Thunder } from "../icons/cloud-lightning.svg";
+
+export default function OutdoorWeatherWidget() {
+  const [weather, setWeather] = useState();
+
+  useEffect(() => {
+    const url =
+      "https://api.weatherapi.com/v1/current.json?key=487e8af21edb4a54b1f142156211105&q=Hamburg&aqi=no";
+
+    fetch(url)
+      .then((res) => res.json())
+      .then((incomingData) => {
+        const newWeather = incomingData.current;
+        setWeather(newWeather);
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Maybe you can alert the error, or redirect to an error page. The user probably won't check the console